interfaces/repository/project_impl.go

```diff
@@ -0,0 +1,128 @@
+package repository
+
+import (
+	"github.com/gofrs/uuid"
+	"github.com/traPtitech/traPortfolio/domain"
+	"github.com/traPtitech/traPortfolio/interfaces/database"
+	"github.com/traPtitech/traPortfolio/interfaces/repository/model"
+	"github.com/traPtitech/traPortfolio/usecases/repository"
+)
+
+type ProjectRepository struct {
+	h database.SQLHandler
+}
+
+func NewProjectRepository(sql database.SQLHandler) *ProjectRepository {
+	return &ProjectRepository{h: sql}
+}
+
+func (repo *ProjectRepository) GetProjects() ([]*domain.Project, error) {
+	projects := []*model.Project{}
```

Comment:
golandだと警告っぽいのが出ていたのでmekeでつくってくれると嬉しいです
```suggestion
	projects := make([]*model.Project, 0)
```